index.html

```diff
@@ -15,11 +15,54 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
+<body class="add-background-color-main">
+
+  <div data-view="entry-form">
+    <form id="entry-form">
+
+      <header class="adjust-nav">
+        <div class="container">Code Journal</div>
+      </header>
```

Comment:
This should be above the `div` with the `data-view` attribute. You want the header to be visible on every view, not just the form view.